lib/kubeclient/common.rb

```diff
@@ -254,6 +241,29 @@ def proxy_url(kind, name, port, namespace = '')
       end
     end
 
+    def get_pod_log(pod_name, namespace = nil, container: nil, previous: false)
+      params = { params: {} }
+      params[:params][:previous] = true if previous
+      params[:params][:container] = container if container
+
+      ns = build_namespace_prefix(namespace || 'default')
```

Comment:
the namespace is not defaulted to 'default' in other methods in the client, and I am thinking why should we start doing it here. If the user is not specifying correctly the namespace, the server will send response accordingly.